examples/receiver/evm/contracts/Setup.sol

```diff
@@ -0,0 +1,35 @@
+// contracts/Implementation.sol
+// SPDX-License-Identifier: Apache 2
+
+pragma solidity ^0.8.0;
+pragma experimental ABIEncoderV2;
+
+import "./Governance.sol";
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Upgrade.sol";
+
+contract Setup is Setters, ERC1967Upgrade {
+    function setup(
+        address implementation,
+        address[] memory initialGuardians,
+        uint16 governanceChainId,
+        bytes32 governanceContract
+    ) public {
+        require(initialGuardians.length > 0, "no guardians specified");
+
+        setOwner(msg.sender);
+
+        Structs.GuardianSet memory initialGuardianSet = Structs.GuardianSet({
+            keys : initialGuardians,
+            expirationTime : 0
+        });
+
```

Comment:
This doesn't set the chain id, which means that it will be initialised to 0. Even if we do want it to be 0, I think we should do it explicitly. I think reserving a high number for it would be better though (it's fine if that number is shared between all receiver contracts on all chains), chain id 0 still has some special meaning which could lead to surprises down the line.

Reply:
What “chain id”?

Reply:
hmm, I guess I got confused when I saw this: https://github.com/certusone/wormhole/pull/1161/files#diff-361bc6909361dc7c00584b3f67d164b2d33e2e8f5808f499bf63f35363c19d4dR35 but indeed it doesn't seem to be mentioned anywhere else. Looking at it closer, the whole interface file is dead code it seems?